src/test/kotlin/uk/gov/justice/digital/hmpps/rediscli/RedisCli.kt

```diff
@@ -11,12 +10,11 @@ object RedisCli {
   fun main(args: Array<String>) {
     Jedis(URI("rediss://localhost:6379")).use { jedis ->
       jedis.auth("xxxx") // use auth_token from the redis secret
-      jedis["foo"] = "bar"
-      val value = jedis["foo"]
-      println(value)
-      println(ArrayUtils.toString(jedis.dump("incident::G8220GL")))
-      jedis.flushDB()
-      // println(jedis.info())
+      // jedis["foo"] = "bar"
+      println(jedis["incident::G1234AB"])
+      println(jedis.dump("incident::G1234AB"))
+      // jedis.flushDB()
+      println(jedis.info())
     }
```

Comment:
slight tidying of this test client - in particular commenting the potentially damaging flush